plugins/woocommerce-blocks/tests/e2e/tests/related-products/related-products.block_theme.spec.ts

```diff
@@ -63,4 +63,54 @@ test.describe( `${ blockData.name } Block`, () => {
 			await editor.getBlockByName( blockData.slug )
 		).toBeVisible();
 	} );
+
+	test( "doesn't display additional Products (Beta) options", async ( {
+		page,
+		admin,
+		editor,
+		editorUtils,
+	} ) => {
+		await admin.visitSiteEditor( {
+			postId: `woocommerce/woocommerce//single-product`,
+			postType: 'wp_template',
+		} );
+		await editorUtils.enterEditMode();
+		const relatedProducts = await editorUtils.getBlockByName(
+			blockData.slug
+		);
+		await editor.selectBlocks( relatedProducts );
+		await editor.openDocumentSettingsSidebar();
+
+		const inspectorControls = page.getByRole( 'region', {
+			name: 'Editor settings',
+		} );
+
+		const upgradeNotice = inspectorControls.getByRole( 'button', {
+			name: 'Upgrade to Product Collection',
+		} );
+		const advancedFilters = inspectorControls.getByRole( 'heading', {
+			name: 'Advanced filters',
+		} );
+
+		await expect( upgradeNotice ).toBeHidden();
+		await expect( advancedFilters ).toBeHidden();
+
+		// Reality check - the above test will pass if:
+		// 1. options are not there - valid case,
+		// 2. selectors changed or are incorrect - invalid case.
+		//
+		// To confirm test is correct, we verify if Products (Beta)
+		// have these options available using the same locators.
+		await editor.setContent( '' );
+		const querySlug = 'core/query';
+		const productsBetaSlug = 'woocommerce/product-query';
+		await editor.insertBlock( {
+			name: querySlug,
+			attributes: { namespace: productsBetaSlug },
+		} );
+		const productsBeta = await editorUtils.getBlockByName( querySlug );
```

Comment:
This should be replaced by `editor`:

```suggestion
		const productsBeta = await editor.getBlockByName( querySlug );
```